core/tx_pool.go

```diff
@@ -133,6 +133,8 @@ type TxPoolConfig struct {
 	Journal   string        // Journal of local transactions to survive node restarts
 	Rejournal time.Duration // Time interval to regenerate the local transaction journal
 
+	SizeLimit uint64 // Maximum size allowed for valid transaction (in KB)
```

Comment:
Would it be worthwhile renaming this to 'TransactionSizeLimit' to make it more descriptive?

Reply:
Agreed. Originally, SizeLimit was part of tx_pool, so it was clearer in that context.